modules/tinymce/src/core/main/ts/ErrorReporter.ts

```diff
@@ -67,14 +67,6 @@ const pluginInitError = (editor: Editor, name: string, err) => {
   displayError(editor, message);
 };
 
-const modelInitError = (editor: Editor, name: string, err) => {
-  // TODO: TINY-8427 Add translation for this
```

Comment:
I guess we should close this Jira now since we're removing this?

Reply:
Yeah, I made a note to do that. I will do that now before I forget

Reply:
Have marked the Jira as `Won't Do` now